src/Swift.Bindings/src/Parser/SwiftABIParser.cs

```diff
@@ -284,15 +284,11 @@ private List<BaseDecl> CollectDeclarations(IEnumerable<Node> nodes, BaseDecl par
                     metadataPtr = DynamicLibraryLoader.invoke(_dylibPath, GetMetadataAccessor(node));
```

Comment:
@stephen-hawley Is it safe to call into dylib at binding time?

Reply:
No it is not.